### PR TITLE
A method bucket.query now sent requests until get last record from se…

### LINKED ISSUE #47: Bug: Bucket.query handle only one HTTP GET request
### What happened?

The method receives the `queryId' but retrieves data for it only once. It must repeat the GET requests until it receives HTTP status 204.

### Version

0.1.0

### OS

Linux, MacOS, Windows

### Snippet to repeat the bug

_No response_

### Logs

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Rumpelshtinskiy**
Closes #47

### Please check if the PR fulfills these requirements

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [x] CHANGELOG.md has been updated (for bug fixes / features / docs)

### What kind of change does this PR introduce?

Bug fix

### What was changed?

The `Bucket.query` return an iterator. 

### Related issues

#47 

### Does this PR introduce a breaking change?

Yes, but it is 0.x version.

### CODE REVIEW

**Comment by Rumpelshtinskiy** [comment]
On testing

File: src/main/java/store/reduct/model/bucket/Bucket.java [hunk 1]
```diff
@@ -283,7 +283,7 @@ public Iterator<Record> query(String entryName, Integer start, Integer stop, Int
      * @throws ReductException
      * @throws IllegalArgumentException
      */
-    Stream<Record> getRecords(@NonNull String entryName, Long queryId) throws ReductException, IllegalArgumentException {
+    Collection<Record> getRecords(@NonNull String entryName, Long queryId) throws ReductException, IllegalArgumentException {
```
Comment (atimin) [inline src/main/java/store/reduct/model/bucket/Bucket.java:286]: you can't return a collection.  If a user requests few gigabytes of data, you run our of memory. It must be an iterator.

**Review by atimin — CHANGES_REQUESTED** [review]

File: src/main/java/store/reduct/model/bucket/Bucket.java (hunk 1 above)
Comment (Rumpelshtinskiy) [reply src/main/java/store/reduct/model/bucket/Bucket.java:286]: Got it

**Comment by atimin** [comment]
@Rumpelshtinskiy  add a record to the changelog please, and I'll merge it.

**Review by atimin — CHANGES_REQUESTED** [review]
```java
public Iterator<Record> query(String entryName, Integer start, Integer stop, Integer ttl) {
```

Timestamps must be long to be used with microseconds.

**Comment by Rumpelshtinskiy** [comment]
Also changed getMetaInfos method, that was broken too

**Pull request merged**